index.html

```diff
@@ -6,14 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Alicante Frontend</title>
+    <meta name="description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
     <link rel="icon" type="image/png" sizes="32x32" href="src/img/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="src/img/favicon-16x16.png">
     <link rel="shortcut icon" href="src/img/favicon.ico">
     <link rel="apple-touch-icon" sizes="180x180" href="src/img/apple-touch-icon.png">
     <link rel="mask-icon" href="src/img/safari-pinned-tab.svg" color="#3a84d0">
     <link rel="manifest" href="src/img/manifest.json">
+    <link rel="stylesheet" href="src/css/style.css">
     <meta name="msapplication-config" content="src/img/browserconfig.xml">
     <meta name="theme-color" content="#3a84d0">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:title" content="Alicante Frontend">
+    <meta name="twitter:description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
+    <meta name="twitter:site" content="@AlicanteFront">
+    <meta name="twitter:creator" content="@AlicanteFront">
+    <meta name="twitter:image:src" content="src/img/alicante-frontend-logo.svg">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Alicante Frontend">
+    <meta property="og:description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
+    <meta property="og:image:width" content="1200">
+    <meta property="og:image:height" content="630">
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:image:secure_url" content="src/img/alicante-frontend-fb.png">
```

Comment:
Facebook is still complaining about not finding the image. It seems like [secure_url](https://developers.facebook.com/docs/sharing/webmasters#images) is just for absolute https paths.

Could you replace `og:image:secure_url` by `og:image`?